### PR TITLE
Complete end to end CI

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

File: provisions/roles/scanner/tasks/main.yml [hunk 1]
```diff
@@ -48,11 +52,20 @@
   with_items:
       - {src: "../beanstalk_worker/cccp-test-worker.service", dest: /etc/systemd/system/cccp-test-worker.service}
 
-- name: Enable Scanner worker
-  service: name="cccp-test-worker" state=restarted enabled=yes
+- name: Skip atomic scan for test environment
+  replace: >
+    dest=/etc/systemd/system/cccp-test-worker.service
+    regexp='worker_start_test.py$'
+    replace='worker_start_test.py ci'
```
Comment (bamachrn) [inline provisions/roles/scanner/tasks/main.yml]: This will require changes in the service file we have for test worker. So that it takes command line args

Comment (rtnpro) [inline provisions/roles/scanner/tasks/main.yml]: @bamachrn why? it's working as it is now.

Comment (bamachrn) [inline provisions/roles/scanner/tasks/main.yml]: Yeah, got it.

**Comment by bamachrn** [comment]
LGTM :+1:

**Pull request merged**